packages/vitest/src/integrations/chai/jest-expect.ts

```diff
@@ -589,11 +589,13 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
     const spy = getSpy(this)
     const spyName = spy.getMockName()
     const pass = spy.mock.results.some(({ type, value: result }) => type === 'return' && jestEquals(value, result))
+    const returns = spy.mock.results.filter(({ type }) => type === 'return').map(({ value: result }) => result).toString()
     this.assert(
       pass,
-      `expected "${spyName}" to be successfully called with #{exp}`,
-      `expected "${spyName}" to not be successfully called with #{exp}`,
-      value,
+      `expected "${spyName}" to be successfully called having return #{exp} at least once`,
+      `expected "${spyName}" to not be successfully called having return #{exp}`,
+      `executions have a successful return value: ${value}`,
```

Comment:
`{}` will return `[Object Object]`

Please, don't reinvent the wheel.

Reply:
try to use chai untils to format message.

Reply:
Please, look at how `toHaveBeenCalledWith` is processing errors. Try to use the same approach, it gives more useful information.

Reply:
May I ask a question?
According the `toHaveBeenCalledWith` which use a util `formatCalls` to specify call stack.
I think a new until such as `formatReturns` should be added to format returns for all calls.
Is this what you want?

Reply:
Yes. It would look basically like `formatCalls`, but uses `returns` instead of `calls`